.github/workflows/windows-qt6.yml

```diff
@@ -27,11 +27,21 @@ jobs:
         with:
           fetch-depth: 2
 
+      - name: check version
+        shell: bash
+        run: |
+          cmake --version || echo "NO CMAKE"
```

Comment:
`which cmake` might also be interesting

Reply:
It was for debugging but I can keep it and add the which command if you want to?